decimal64math_test.go

```diff
@@ -379,3 +379,93 @@ func BenchmarkDecimal64Sub(b *testing.B) {
 		_ = x[i%len(x)].Sub(y[i%len(y)])
 	}
 }
+
+func TestAddOverflow(t *testing.T) {
```

Comment:
Let's also test underflow scenarios for Add/Quo, etc. (Min/2 = 0)

Reply:
Added as [comment](https://github.com/anz-bank/decimal/issues/1#issuecomment-456246840) to #1 - let's get this PR merged.